client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -54,47 +57,52 @@ class DeployedModel:
         # <DeployedModel 01234567-0123-0123-0123-012345678901>
 
     """
-    def __init__(self, _host=None, _run_id=None, _from_url=False, **kwargs):
-        # this is to temporarily maintain compatibility with anyone passing in `socket` and `model_id` as kwargs
-        # TODO: instate `host` and `run_id` params
-        # TODO: remove the following block of param checks
+    def __init__(self, host=None, run_id=None, creds=None, access_token=None, prediction_url=None, **kwargs):
```

Comment:
Since `from_url()` has its params ordered `token` then `creds`, would you mind if we put `access_token` and `creds` in the same order here?